d3a_api_client/redis.py

```diff
@@ -9,7 +9,7 @@
 
 
 root_logger = logging.getLogger()
-root_logger.setLevel(logging.ERROR)
+root_logger.setLevel(logging.INFO)
```

Comment:
I think it is better to revert it by default to ERROR, and to let the user decide if they want to use a more informative log level

Reply:
Yes, you are right. forgot to change it back